PlayerMethods.h

```diff
@@ -2187,6 +2187,26 @@ namespace LuaPlayer
 #endif
         return 0;
     }
+    
+    int UnbindAllInstances(Eluna* /*E*/, lua_State* L, Player* player)
+    {
+        for (uint8 i = 0; i < MAX_DIFFICULTY; ++i)
+        {
+            Player::BoundInstancesMap& binds = player->GetBoundInstances(Difficulty(i));
+            for (Player::BoundInstancesMap::iterator itr = binds.begin(); itr != binds.end();)
+            {
+                if (itr->first != player->GetMapId())
+                {
+                    DungeonPersistentState* save = itr->second.state;
+                    std::string timeleft = secsToTimeString(save->GetResetTime() - time(NULL), true);
```

Comment:
save and timeleft are unused, should remove them.
Otherwise seems that this is identical to TC and should work on all cores(TC/CMa/Ma) most likely.
(wouldnt swear on all patches)